easybuild/easyblocks/p/pytorch.py

```diff
@@ -253,7 +255,31 @@ def test_step(self):
             'python': self.python_cmd,
             'excluded_tests': ' '.join(excluded_tests)
         })
-        super(EB_PyTorch, self).test_step()
+
+        (out, ec) = super(EB_PyTorch, self).test_step(return_output_ec=True)
+        if ec:
+            print_warning("Test command had non-zero exit code (%s)!" % ec)
```

Comment:
I would move that down so it is only printed when the "X tests failed" is NOT printed. And then possibly adjust the message. Currently this is over-verbose:
> Test command had non-zero exit code
> 10 test (out of 20) failed: ...
> Too many failed tests ...

Reply:
fixed in 82011bf